egs/librispeech/ASR/zipformer/zipformer.py

```diff
@@ -1305,6 +1305,12 @@ def __init__(
     ) -> None:
         """Construct a CompactRelPositionalEncoding object."""
         super(CompactRelPositionalEncoding, self).__init__()
+        if torch.jit.is_tracing:
+            # 10k frames correspond to ~100k ms, e.g., 100 seconds, i.e., 
```

Comment:
The previous comment from the master branch is wrong. Could you please fix it?
```
# It assumes that the maximum input, after downsampling, won't have more than 
# 10k frames.

# The first downsampling factor is 2, so the maximum input
should contain less than 20k frames, e.g., less than 200 seconds, i.e., 3.33 minutes
```

Reply:
OK